.github/workflows/build.yml

```diff
@@ -14,21 +14,26 @@ jobs:
   build:
     name: Maven build
     runs-on: ubuntu-18.04
-
     strategy:
       matrix:
         maven-version:
           - '3.3.9'
           - '3.6.1'
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           fetch-depth: 2
       - name: Set up JDK
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
+        with:
+          java-version: 8
+          distribution: 'zulu'
```

Comment:
Since this change is for optimization, it'd be better to use the `temurin` JDK distribution as this is in the [Hosted Tool Cache](https://github.com/actions/setup-java/blob/v3.0.0/docs/advanced-usage.md#hosted-tool-cache) of the [ubuntu-18.04 runner](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-Readme.md#java) (it'd avoid having to download and untar the JDK).

Reply:
6f6a486622eb3952e4a92efc645e9b0dd12d61d0